tyson/README.md

```diff
@@ -1,26 +1,68 @@
 # TySON 🥊
-### Use TypeScript as a configuration language
+### TypeScript as a Config Language
```

Comment:
Spell out `Configuration`:
`TypeScript as a Configuration Language`

Because `Configuration Language` is the term being used in the space. See https://dhall-lang.org/ or https://jsonnet.org/